### PR TITLE
Translate 2020 CSS chapter to Portuguese

### PR DESCRIPTION
**Pull request by emanuelgsouza**
Makes progress on #505

### CODE REVIEW

File: src/content/pt/2020/css.md [hunk 1]
```diff
@@ -0,0 +1,1738 @@
...
+  sheets_gid="863628849",
+  sql_file="top_selector_classes_wp_fa_prefixes.sql"
+) }}
+
+### IDs
+
+Apesar dos IDs serem desencorajados atualmente em alguns círculos devido à sua grande especificidade, a maioria dos sites ainda os utiliza, embora com parcimônia. Menos da metade das páginas utilizava mais de um ID em qualquer um de seus seletores (tinha uma especificidade máxima de (1,x,y) ou menos) e quase todos tinham uma especificidade média que não incluía IDs (0,x,y). Consulte a [especificação de seletores da W3C](https://www.w3.org/TR/selectors/#specificity-rules) para obter mais detalhes de como calcular especificidade e esta notação (a,b,c).
+
+Mas para que os IDs são usados? Acontece que os IDs mais populares são estruturais: `#content`, `#footer`, `#header`, `#main`, embora existam [elementos HTML correspondentes](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Document_and_website_structure#HTML_layout_elements_in_more_detail) que poderiam ser utilizados como seletores ao mesmo tempo em que melhoram a marcação semântica.
```
Comment (bazzadp) [inline src/content/pt/2020/css.md:125]:
```suggestion
Mas para que os IDs são usados? Acontece que os IDs mais populares são estruturais: `#content`, `#footer`, `#header`, `#main`, embora existam [elementos HTML correspondentes](https://developer.mozilla.org/pt-BR/docs/Learn/HTML/Introduction_to_HTML/Document_and_website_structure#elementos_de_layout_do_html_em_mais_detalhes) que poderiam ser utilizados como seletores ao mesmo tempo em que melhoram a marcação semântica.
```

File: src/content/pt/2020/css.md [hunk 2]
```diff
@@ -0,0 +1,1738 @@
...
+  caption="Popularidade relativa de comprimentos zerados por unidade pela porcentagem das ocorrências em páginas para celular.",
+  description="Gráfico de pizza mostrando comprimentos zerados sem unidade (AKA unitless) utilizados em 88,7% do tempo em páginas para celular, a unidade `px` em 10,7%, e outras unidades em 0,5% dos casos.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRpe_HsNGpekn6YZV9k6QGmcZPxalqnDrL7DrDY-7X65RZEf_-aGfWuEvhk-yWV83ctIceE1bppCLpj/pubchart?oid=1935151776&format=interactive",
+  sheets_gid="313150061",
+  sql_file="units_zero.sql"
+) }}
+
+### Cálculos
+
+Quando a função [`calc()`](https://developer.mozilla.org/pt-BR/docs/Web/CSS/calc()) foi introduzida para realizar cálculos entre diferentes unidades no CSS, foi uma revolução. Anteriormente, apenas os pré-processadores eram capazes de realizar tais cálculos, mas os resultados eram limitados a valores estáticos e não confiáveis, uma vez que faltava o contexto dinâmico que muitas vezes é necessário.
```
Comment (bazzadp) [inline src/content/pt/2020/css.md:439]:
```suggestion
Quando a função [`calc()`](https://developer.mozilla.org/pt-BR/docs/Web/CSS/calc()) foi introduzida para realizar cálculos entre diferentes unidades no CSS, foi uma revolução. Anteriormente, apenas os pré-processadores eram capazes de realizar tais cálculos, mas os resultados eram limitados a valores estáticos e não confiáveis, uma vez que faltava o contexto dinâmico que muitas vezes é necessário.
```

File: src/content/pt/2020/css.md [hunk 3]
```diff
@@ -0,0 +1,1738 @@
...
+- Um erro de digitação que foi encontrado em uma biblioteca popular. Por meio dessa análise, constatamos que a propriedade `white-wpace` estava presente em 234.027 sites. São sites demais para que o mesmo erro de digitação tenha ocorrido organicamente, então decidimos dar uma olhada nele. E eis que [descobriu-se](https://twitter.com/rick_viscomi/status/1326739379533000704) qie era o widget do Facebook! A correção já existe.
+- E outra estranheza: a propriedade `font-rendering` aparece em 2.575 páginas. No entanto, não podemos encontrar evidências da existência de tal propriedade, com ou sem um prefixo. Existe o não padrão [`-webkit-font-smoothing`](https://medium.com/better-programming/improving-font-rendering-with-css-3383fc358cbc) que é extremamente popular, aparecendo em 3 milhões de sites, ou cerca de 49% das páginas, mas `font-rendering` não é suficientemente próximo para ser um erro de ortografia. Existe [`text-rendering`] (https://developer.mozilla.org/en-US/docs/Web/CSS/text-rendering) que é usado em cerca de 100K de sites, portanto, é concebível que 2,5K e todos os desenvolvedores se lembraram erroneamente e cunharam um portmanteau de `font-smoothing` e `text-rendering`.
```
Comment (bazzadp) [inline src/content/pt/2020/css.md:1648]:
```suggestion
- E outra estranheza: a propriedade `font-rendering` aparece em 2.575 páginas. No entanto, não podemos encontrar evidências da existência de tal propriedade, com ou sem um prefixo. Existe o não padrão [`-webkit-font-smoothing`](https://medium.com/better-programming/improving-font-rendering-with-css-3383fc358cbc) que é extremamente popular, aparecendo em 3 milhões de sites, ou cerca de 49% das páginas, mas `font-rendering` não é suficientemente próximo para ser um erro de ortografia. Existe [`text-rendering`](https://developer.mozilla.org/pt-BR/docs/Web/CSS/text-rendering) que é usado em cerca de 100K de sites, portanto, é concebível que 2,5K e todos os desenvolvedores se lembraram erroneamente e cunharam um portmanteau de `font-smoothing` e `text-rendering`.
```

**Review by bazzadp — CHANGES_REQUESTED** [review]
Not an easy chapter to edit as it's our biggest one!

A few small corrections but other than that looks good to me from a technical front.

**Comment by emanuelgsouza** [comment]
Hey @bazzadp , I implemented the requested changes. Could you take a look again? Thank you for your review.

**Review by bazzadp — APPROVED** [review]

**Comment by bazzadp** [comment]
All good to merge from my side.

@HTTPArchive/portuguese-translators if anyone is able to review this let me know. Otherwise I'll merge in next few days.

**Pull request merged**